### PR TITLE
Certificate faqs

### PR DESCRIPTION
**Pull request by weppos**
This PR adds a list of common certificate FAQs.

---

Related to #173

### CODE REVIEW

**Comment by weppos** [comment]
@dnsimple/dnsimple can you please proofread the article?

File: content/articles/faq-ssl-certificates.markdown [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+1.  #### I want to purchase an SSL certificate. What is the DNSimple subscription?
+
+    DNSimple is a subscription-based service. In order to access our domain services, you need an active subscription to our platform.
+
+    The subscription gives you access to both the included platform services (such as the API access and the DNS hosting), and the paid products (such as the domain registration, SSL certificates, email forwarding, etc.).
+
+1.  #### I want to purchase an SSL certificate. Do I need a subscription?
+
+    Yes. As explained in our previous FAQ, the SSL certificates are one of our paid products, provided s part of our subscription services.
```
Comment (tbunnyman) [inline content/articles/faq-ssl-certificates.markdown]: provided _as_ part of...
I would also make subscription service singular.

File: content/articles/faq-ssl-certificates.markdown [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    In order to purchase an SSL certificate you need an account active subscription at the time of the purchase. You are not required to have an active subscription to use or install the certificate on your domain/server, however please note that if you disable the subscription you will not receive any additional service for the domain or the certificate.
+
+    You will also need a valid, active subscription at the time you will need to renew the certificate, or if you want to reissue the certificate.
```
Comment (tbunnyman) [inline content/articles/faq-ssl-certificates.markdown]: I would say "You will also need a valid, active subscription when you renew the certificate..."

File: content/articles/faq-ssl-certificates.markdown [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+</div>
+
+## Domains and SSL certificates
+
+<div class="section-faq" markdown="1">
+1.  #### Do I need to transfer the domain to you to purchase an SSL certificate?
```
Comment (tbunnyman) [inline content/articles/faq-ssl-certificates.markdown]: I would say "to your account" instead "to you"

File: content/articles/faq-ssl-certificates.markdown [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+    No. You can purchase an SSL certificate for a domain that is not registered with us, without transferring it to us. However, you will need to add the domain to your account in order to be able to request the certificate. You also need a valid, active subscription.
```
Comment (tbunnyman) [inline content/articles/faq-ssl-certificates.markdown]: The first comma can be removed

File: content/articles/faq-ssl-certificates.markdown [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+    Just follow the general instruction explained in the [Getting Started](/articles/getting-started-ssl-certificates/) guide. When you are asked to add the domain to purchase the certificate, select the option to add a domain without transferring or registering it.
```
Comment (tbunnyman) [inline content/articles/faq-ssl-certificates.markdown]: comma can be removed. This is a toss up actually, it feels unnecessary in flow but grammatically is not incorrect to have

File: content/articles/faq-ssl-certificates.markdown [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+## Prices and Subscription
+
+<div class="section-faq" markdown="1">
+1.  #### What is the price of an SSL certificate?
+
+    The price depends on the certificate type. To learn more visit the [SSL Certificates](/articles/ssl-certificates/) article.
+
+1.  #### I want to purchase an SSL certificate. What is the DNSimple subscription?
+
+    DNSimple is a subscription-based service. In order to access our domain services, you need an active subscription to our platform.
```
Comment (meeunier) [inline content/articles/faq-ssl-certificates.markdown]: DNSimple is a subscription-based service. You need an active subscription in our to use our platform.

File: content/articles/faq-ssl-certificates.markdown [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+    No. You can purchase an SSL certificate for a domain that is not registered with us, without transferring it to us. However, you will need to add the domain to your account in order to be able to request the certificate. You also need a valid, active subscription.
+
+    Just follow the general instruction explained in the [Getting Started](/articles/getting-started-ssl-certificates/) guide. When you are asked to add the domain to purchase the certificate, select the option to add a domain without transferring or registering it.
+
+1.  #### Do I need to host the domain with you to purchase an SSL certificate?
+
+    No. You can purchase an SSL certificate for a domain that is not hosted with us. However, you will need to add the domain to your account in order to be able to request the certificate. You also need a valid, active subscription.
+
+    Just follow the general instruction explained in the [Getting Started](/articles/getting-started-ssl-certificates/) guide. When you are asked to add the domain to purchase the certificate, select the option to add a domain without transferring or registering it.
```
Comment (tbunnyman) [inline content/articles/faq-ssl-certificates.markdown]: Same comment as previous; comma can be removed. This is a toss up, it feels unnecessary in flow but grammatically is not incorrect to have

File: content/articles/faq-ssl-certificates.markdown [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+<div class="section-faq" markdown="1">
+1.  #### What type of SSL certificates does DNSimple provide?
+
+    We currently offer different types of _domain-validated_ SSL certificates, signed by globally recognized certificate authorities. We don't provide _organization-validated_ (OV) or _extended-validation_ (EV).
+
+    To learn more visit the [SSL Certificates](/articles/ssl-certificates/) article.
+
+1.  #### What certificates show the company name in the green bar near the padlock?
+
+    The green bar in the browser is displayed only if you purchase an _extended-validation_ (EV) certificate. We currently don't sell EV certificates.
```
Comment (tbunnyman) [inline content/articles/faq-ssl-certificates.markdown]: I would not use a contraction here. "We currently do not sell EV certificates" it sounds more official.

File: content/articles/faq-ssl-certificates.markdown [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+
+    We currently only sell DV certificates, that are less expensive than EV certificates and offer the same cryptographic level of security. If you still need/want to purchase an EV certificate, we provide some recommendation at [this page](/articles/can-ev-ssl-certificates/).
```
Comment (tbunnyman) [inline content/articles/faq-ssl-certificates.markdown]: ..DV certificates. These are less expensive...
an EV certificate~~,~~ we provide

**Pull request merged**